src/store/reducers/index.js

```diff
@@ -112,6 +127,7 @@ export const reducer = (
 			// TODO we need to set focused: true and search for the currently focused block and
 			// set that one to `focused: false`.
 			blocks.push(action.block);
+			insertBlock(blocks, action.block, action.clientIdAbove);
```

Comment:
Hmm, should this `insertBlock` replace the `blocks.push` in the previous line?

Reply:
ooops thank Lord you catched that, addressed c1b51b9